src/Traits/Transformable.php

```diff
@@ -85,11 +85,11 @@ public function lowercase(string $mode = Config\Lowercase::ALL) : self
         Config\Lowercase::validateOption($mode);
 
         if ($mode == Config\Lowercase::WORDS) {
-            $words = array_map(function ($word) {
-                return lcfirst($word);
-            }, explode(' ', $this->string));
+            $string = preg_replace_callback('/(^|\s)(\w)/', function ($matched) {
```

Comment:
How about using `[^|\s](\w+)` for the regex pattern. We can then use `lcfirst()` on `$matched[1]` below which feels more accurate to me.

Reply:
`[^|\s]` won't work.  It will match a negated set of `|` and `\s`, if you want to stick with `lcfirst`, we can do:

```php
            $string = preg_replace_callback('/(^|\s)(\w)/', function ($matched) {
                return $matched[1].lcfirst($matched[2]);
            }, $this->string);
```

Reply:
I was just laying in bed thinking about how I got that wrong. :grin:  However, my mistake led me to think of this pattern: `([A-Z][^\s]*)` which will capture any string of characters starting with a capital letter followed by zero or more characters that are not a space, tab or newline. Not only will this allow us to use `lcfirst()` on `$matched[1]` like I was hoping but it will also reduce the results being processed helping with performance of long strings.

Reply:
Very nice.  Works perfectly.  I have updated the commit with the change, and added a comment for clarity.